CHANGELOG.md

```diff
@@ -180,6 +180,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Fixed truncated backprop through time enablement when set as a property on the LightningModule and not the Trainer ([#8804](https://github.com/PyTorchLightning/pytorch-lightning/pull/8804/))
 
+- Fixed avoid wrapping LightningModule with data-parallel when not fitting in `DDPPlugin`, `DDPSpawnPlugin`, `DDPShardedPlugin`, `DDPSpawnShardedPlugin` ([#6977]https://github.com/PyTorchLightning/pytorch-lightning/issues/6977).
+
```

Comment:
i think these changes were meant for another PR?

Reply:
yeah. the first iteration of the PR somehow mixed with another [PR](https://github.com/PyTorchLightning/pytorch-lightning/pull/8632). I've rebased and moved unrelated commits out.